src/ui/common/src/components/pages/workflow/id/index.tsx

```diff
@@ -439,6 +442,33 @@ const WorkflowPage: React.FC<WorkflowPageProps> = ({
 
   const drawerHeaderHeightInPx = 64;
 
+  const handleTabChange = (event: React.SyntheticEvent, newTab: string) => {
+    setCurrentTab(newTab);
+  };
+
+  interface TabPanelProps {
+    children?: React.ReactNode;
+    index: string;
+    value: string;
+  }
+
+  // TODO: Make this a component, probably can put this near the other tab component that we have
```

Comment:
Reminder to create a Linear issue for each TODO.

Reply:
done, can find here:
https://linear.app/aqueducthq/issue/ENG-2409/tabpanel-component